src/rpc/blockchain.cpp

```diff
@@ -1161,6 +1163,162 @@ static RPCHelpMan pruneblockchain()
     };
 }
 
+static RPCHelpMan dumpcoinstats()
+{
+    return RPCHelpMan{"dumpcoinstats",
+                "\nDumps content of coinstats index to a CSV file.\n",
```

Comment:
nit:

```suggestion
        "Dumps content of coinstats index to a CSV file.",
```

Can remove some whitespace and `\n`.